core/src/main/java/com/linecorp/armeria/common/AbstractRequestContextBuilder.java

```diff
@@ -200,7 +201,7 @@ public AbstractRequestContextBuilder meterRegistry(MeterRegistry meterRegistry)
      */
     protected final EventLoop eventLoop() {
         if (eventLoop == null) {
-            eventLoop = CommonPools.workerGroup().next();
+            eventLoop = new DefaultEventLoop();
         }
         return eventLoop;
     }
```

Comment:
Let's make this method return `eventLoop` as is.
```java
@Nullable
protected final EventLoop eventLoop() {
    return eventLoop;
}
```

Reply:
Then it will causes a NullPointerException in many places that call `eventloop()`.
I'll think a little more about what I'm missing! Thanks 🙇